### PR TITLE
Add yandex images as image identification option

### PR DESCRIPTION
**Pull request by haruhianon609**
**NOTE: This is compiltely untested!** I suggest that someone tests that this works before merging the PR.

### CODE REVIEW

**Comment by 27chan** [comment]
The best way to use Yandex reverse search is to use it like this.

`<a rel="noreferrer" href="https://yandex.com/images/search?url={{ config.domain }}{{ config.uri_img }}{{ file.file }}&rpt=imagelike" target="_blank">Yandex</a>`

So we can find similar images, I use it in [bazukachan](https://github.com/27chan/bazukachan) and I have no problems besides having better results.

**Comment by haruhianon609** [comment]
ok, modified

File: templates/post/image_identification.html [hunk 1]
```diff
@@ -13,5 +13,8 @@
 		{% if config.image_identification_iqdb %}
 			<a href="http://iqdb.org/?url={{ config.domain }}{{ config.uri_img }}{{ file.file }}" target="_blank">iqdb</a>
 		{% endif %}
+		{% if config.image_identification_yandex %}
+                        <a rel="noreferrer" href="https://yandex.com/images/search?url={{ config.domain }}{{ config.uri_img }}{{ file.file }}&rpt=imagelike" target="_blank">Yandex</a>
+                {% endif %}
```
Comment (ctrlcctrlv) [inline templates/post/image_identification.html:18]: Fix indentation.

**Review by ctrlcctrlv — CHANGES_REQUESTED** [review]

File: templates/post/image_identification.html (hunk 1 above)
Comment (ctrlcctrlv) [reply templates/post/image_identification.html]: Did it for you.

**Pull request merged**